### ISSUE TITLE
When the scummvm repository updates and is pulled locally, 'make' on this repository build dir builds incorrectly

### ISSUE THREAD

**Issue by i30817** [diablodiab/libretro-scummvm-backend#5]
What happens is that make says it built the new changes, and then fails at runtime with a missing or incorrect symbol (from the new upstream code).

The obvious solution is, of course, call make clean before if the parent repository had a update.

However, i believe it would be valuable to automate this so that the build make file _here_ can handle this cause automatically, because it makes automation complicated otherwise, or waste a lot of cpu and electricity rebuilding sub-modules it doesn't need by always calling make clean.

I don't know the _best_ way to do this, if it's better to make this a submodule of the main project or what, just reporting.

**Comment by i30817** [comment]
[This might help](https://github.com/libretro/libretro-uae/commit/e2fff5cc4d4e68338d83e2371b4ab13b6bd2a79c), although i'm not sure it's the same issue.

**Comment by ner00** [comment]
I seem to have the same "issue", @i30817 did you ever resolve this to prevent cleaning and rebuilding after pulling?

**Comment by i30817** [comment]
No, i didn't manage to solve it. To tell the truth, i didn't even try.

**Comment by ner00** [comment]
I was too lazy to try your suggestion above - that's why I asked - ahah
